packages/create-snap-cli/README.md

```diff
@@ -0,0 +1,28 @@
+# @metamask/create-snap
+
+A CLI for creating MetaMask Snaps.
+
+## Usage
+
+Use this command line tool directly with `yarn create` or `npx` to create a new snap project:
+
+```sh
+yarn create @metamask/snap your-snap-name
+# or...
+npx @metamask/create-snap your-snap-name
```

Comment:
```suggestion
yarn create @metamask/create-snap your-snap-name
# or...
npx @metamask/create-snap your-snap-name
```

Reply:
No. The way `yarn create` works with namespaced packages is you remove the word `create` from the package name.